docs/pages/settings-gentime.md

```diff
@@ -70,3 +70,13 @@ settings.change({
 ```
 
 It is considered idiomatic to use the Nexus Prisma configuration file instead of inline generator block configuration. Inline generator block configuration lacks autocomplete and inline JSDoc. The only reason `output` is supported is to be [symmetrical with Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client#the-location-of-prisma-client) and thus ease onboarding.
+
+If you added a custom output path because the output will be in `.js` you will need to include the `.js` files into your `tsconfig.ts` file by adding this prop.
+
+```json
+{
+  "compilerOptions": {
+    "allowJs": true
+  }
+}
+```
```

Comment:
I am not against documenting things in the website but it should always be in addition to inline contextual feedback. So in this case:

1. When user uses this setting we should read their tsconfig to check if they are properly setup or not. Reading the tsconfig needs to be done with the TS compiler because simply reading the JSON file is not enough, in cases of config extends etc. If the user's project is being built with custom tsconfig file name there needs to be a way for them to tell Nexus Prisma about it. There will be multiple lint-like use-cases for analyzing the user's tsconfig. So it makes sense that the user can tell Nexus Prisma once where the tsconfig file is and then from there NP is doing all the current and future analysis checks.

3. If they are not properly setup we could output into the terminal a warning explaining the problem and the fix. Furthermore we could auto-fix the issue by writing the necessary config to their tsconfig.

4. Even if the user is properly setup, there should still be terminal output during the generation step that tells them about the problem and the solution and that they are correctly configured. A user for example might have `allowJs` set true for _other_ reasons and not even realize NP needs it in this case.

5. This lint action needs to be configurable so that the user can:
	1. Disable autofix (default on)
	2. Disable the rule completely so that there are no checks done by NP (default on)
	3. Disable learning mode (mentioned in (4) above) for users that know what is going on. (default on)